src/RadioGroup.js

```diff
@@ -12,7 +12,7 @@ class RadioGroup extends React.Component {
         }),
         container: PropTypes.string,
         name: PropTypes.string.isRequired,
-        onChange: PropTypes.func.isRequired,
+        onChange: PropTypes.func,
```

Comment:
Yeah the only reason why I see onChange not beeing required is just for prototyping because otherwise you have no way to get the value from the input.